### ISSUE TITLE
less2css.org is down

### ISSUE THREAD

**Issue by scottrippey** [bfricka/less-preview#29]
Long time, no see!  Hope you're doing well.
Just wanted to make you aware that the website is unreachable.  
Not sure if you put much time into this anymore, but I wanted to let you know it was down.

**Comment by lukeapage** [comment]
If not maybe we can move this to github pages off the less organisation?

**Comment by scottrippey** [comment]
@lukeapage I know there were a number of server-side components.  The Less releases were automatically pulled down from GitHub, for example.  So, we'd need to refactor those to use a manageable hard-coded list.  
And we'd need @bfricka to update his domain name, or just use the less.github.io/less2css or something.

**Comment by bfricka** [comment]
Sorry been off the grid for a couple days! Should be back up now. Amazon decided to switch something on me and shut down the instance (they changed the public IP as well).

@lukeapage I don't have any issues bringing this into less org. That's where it belongs. Let me know what your overall thoughts are, because I trust you and I'm amenable to anything that makes sense.

Right now it just sits on a free EC2 instance. It runs using `forever` and just sits behind nginx.

**Comment by bfricka** [comment]
@scottrippey I actually do have code that does the automatic version checking and all that in the "Overhaul" branch, but, embarrassingly, I haven't merged it b/c I wanted to do some work on getting it compatible w/ less 2+ first :(

And obviously that sorta disappeared from my consciousness until now...

**Comment by scottrippey** [comment]
@bfricka Side question: Does the free EC2 instance run Node?  I was looking for Node hosting, but couldn't find it cheap, and I wasn't sure if Amazon supported it.

**Comment by scottrippey** [comment]
Also, I'm trying http://www.less2css.org to no avail ...

**Comment by bfricka** [comment]
Super lame :P I'll try to diagnose really quick but I'll probably have to look at it in more detail this evening.

Re: your first question. Yes EC2 free tier lasts for like a year or so and it's cheap thereafter. It basically just gives some resources to install whatever you want, and has a community of images for various setups. Mine just runs Ubuntu LTS and I installed Node myself.

**Comment by lukeapage** [comment]
@bfricka 
I would probably prefer not having any server-side components at all - you can have a compile step in github pages and I'd have that fetch all the less.js versions and store them in a directory and have the client download as necessary.
But I'm not sure how much time I can spare or how much this departs from what you've done - I have never really looked.

I guess our frustration is that its a great site, but it goes down quite often and doesn't have version 2 support (How far are you with that, anything I can do?)

**Comment by scottrippey** [comment]
Site back up :)

**Comment by bfricka** [comment]
Hey @lukeapage, I agree with you. I mainly did the server-side stuff because I wanted to play around with it and learn about it. For awhile there it really was unstable, but over the past few months (with the exception of this outage), it's been super stable. That said, I've had my fun, and I'd prefer to move on to other things.

As far as less 2, I think I need to just go through and see the changelog to see if there's anything I really have to worry about, although at first glance I didn't see anything jump out at me.

I haven't looked at the codebase in awhile, but when I previously have, I noticed that finding all options that less can take is a bit awkward. It would be really nice if there were an "options.js" or something similar (like traceur, and other libs have). The operation of this front-end in its most basic form, relies on normalized options, instantiation, and usage of the `less` class. That's about it.

So to sum up, I'm A-okay with moving this into the `less` org and moving it to GH pages. I'll have to read up on the compile step (is it ruby, shell, js?). I think that's the first goal. Get it running as a front-end using GH-pages and move it to `less` org, then deal w/ less 2.

Thoughts?
